FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthProvider.swift

```diff
@@ -199,7 +199,7 @@ import Foundation
       }
 
       let recaptchaVerifier = AuthRecaptchaVerifier.shared(auth: auth)
-      try await recaptchaVerifier.retrieveRecaptchaConfig(forceRefresh: false)
+      try await recaptchaVerifier.retrieveRecaptchaConfig(forceRefresh: true)
```

Comment:
Why are we changing this? Rest LGTM

Reply:
In our old design, we decided to do a lazy fetch here given reCAPTCHA team has a quota on their side.  After we looked at some of metrics, we identified that phone auth has a lower frequency than restarting the app (force fetch) anyway.  Hence we reverting this logic.  A future plan would be either ask rCE team to increase the quota or us storing this value in a more permanent location on apps.